core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -660,7 +662,7 @@ private[kafka] class Acceptor(val endPoint: EndPoint,
       serverChannel.socket().setReceiveBufferSize(recvBufferSize)
 
     try {
-      serverChannel.socket.bind(socketAddress)
+      serverChannel.socket.bind(socketAddress, listenBacklogSize)
```

Comment:
Could we add a unit test which verifies that the configuration is correctly applied to the socket?

Reply:
Good point.
Let me try that.

Reply:
@dajac Added a test. Please take a look.